### PR TITLE
[FIX] Fix issue #86

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Comment by ghost** [comment]
:+1:

File: mgmtsystem_nonconformity/model/mgmtsystem_nonconformity.py [hunk 1]
```diff
@@ -203,8 +204,9 @@ def case_send_note(self, cr, uid, ids, text, data=None, context=None):
                 o = self.browse(cr, uid, ids, context=context)[0]
                 post = '''
```
Comment (pedrobaeza) [inline mgmtsystem_nonconformity/model/mgmtsystem_nonconformity.py]: Why not simply put `u'''` here?

**Comment by LeartS** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**